webapp/src/Form/Type/ContestType.php

```diff
@@ -24,59 +24,75 @@ class ContestType extends AbstractExternalIdEntityType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $this->addExternalIdField($builder, Contest::class);
-        $builder->add('shortname', TextType::class);
-        $builder->add('name', TextType::class);
+        $builder->add('shortname', TextType::class, [
+            'help' => 'Contest name as shown in the top right.'
+        ]);
+        $builder->add('name', TextType::class, [
+            'help' => 'Contest name in full as shown on the scoreboard.'
+        ]);
         $builder->add('activatetimeString', TextType::class, [
             'label' => 'Activate time',
+            'help' => 'Time when the contest becomes visible for teams. Must be in the past to enable submission of jury submissions.',
         ]);
         $builder->add('starttimeString', TextType::class, [
             'label' => 'Start time',
+            'help' => 'Absolute time when the contest starts.',
         ]);
         $builder->add('starttimeEnabled', ChoiceType::class, [
-            'label' => 'Start time enabled',
+            'label' => 'Start time countdown enabled',
             'expanded' => true,
             'choices' => [
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'Disable to delay the contest start and stop the countdown. Enable again after setting a new start time.',
+            # What happens when the countdown is disabled, but the start time is in the past/time 'crosses' the start time?
```

Comment:
In that case, we still do the same and say "start delayed" and disallow looking at problems/submissions:
![image](https://user-images.githubusercontent.com/418721/98850097-8872a680-2454-11eb-9351-70e8ef91c5ea.png)

We may want to do just forbid this (in a separate commit), but for now just remove this line.